package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.8.1",
```

Comment:
I will make another PR to bump the version since it's currently alpha.133, but my PR is 132.

Reply:
since we already have an alpha for a later PR number, maybe it would make more sense to merge this PR without a version bump and open a new PR for that so the numbering remains consistent